Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    before(:each) do
+      if described_class.instance_variable_defined?(:@svn)
+        described_class.send(:remove_instance_variable, :@svn)
+      end
+    end
+
+    it "returns svn version if svn available" do
+      expect(described_class.svn_available?).to be_truthy
+    end
+
+    it "returns svn version if already set" do
```

Comment:
True. I guess I should remove it. I've removed such similar tests in git tests as well.  Thanks :)